java/src/main/java/com/google/appengine/tools/pipeline/impl/servlets/PipelineServlet.java

```diff
@@ -99,7 +108,7 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp)
    * This must match the URL in web.xml
    */
   public static String baseUrl() {
-    String baseURL =  System.getProperty(BASE_URL_PROPERTY, "/_ah/pipeline/");
+    String baseURL = ConfigProperty.BASE_URL_PROPERTY.getValue( "/_ah/pipeline/");
```

Comment:
i find this syntax rather confusing ... `get().orElse("/_ah/pipeline/")` is more readable to me.